README.md

```diff
@@ -242,3 +242,71 @@ Testing looks something like this:
 
 But in production use you'd expect it more like
    `curl <auth to your proxy> https://ssoproxy.example.com/enroll_user -d @~/.ssh/bob.pub`
+
+### Github Support
+
+Sharkey supports issuing certificates for SSH access to Github by mapping SAML identities to Github usernames through 
```

Comment:
It might be worth to add a bit of context here, e.g.:

"
Sharkey supports issuing user certificates that are compatible with GitHub SSH CA format by:
- mapping a GitHub username to a SAML identity, and
- including appropriate GitHub username in each certificate

GitHub [supports](https://github.blog/2019-08-14-ssh-certificate-authentication-for-github-enterprise-cloud/) authentication using SSH certificates for Enterprise Cloud accounts. The only requirement is that certificates [include GitHub usernames](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-ssh-certificate-authorities), so that they can be matched to a particular user.

Sharkey already requires SSO proxy for the user certificate feature. Additionally, the GitHub integration requires that the GitHub organization is configured with SSO (i.e. non-GitHub) access.
"